File: java/armstrong.java
import java.util.*;
class armstrong
{
    public static void main(String args[])
    {
        int num,result=0,rem;
        System.out.println("enter the number:");
        Scanner sc=new Scanner(System.in);
        num=sc.nextInt();
        int temp=num;
        while(num>0)
        {
            rem=num%10;
            result=result+(rem*rem*rem);
            num=num/10;
        }
        if(result==temp)
        System.out.println("armstrong");
        else 
        System.out.println("not armstrong");
    }
}

File: c/armstrong.c
#include<stdio.h>
int main()
{
    int num,org_num,remainder,result=0;
    printf("Enter the number: ");
    scanf("%d",&num);
    org_num = num;

    while(org_num!=0){
        remainder = org_num%10;
        result = result+(remainder * remainder * remainder);

        org_num = org_num/10;
    }

    if(org_num==num){
        printf(num,"Its armstrong number.");
    }
    else{
        printf(num,"Its not an armstrong number.");
    }
    return 0;

}